tests/compiler/CMakeLists.txt

```diff
@@ -59,3 +59,20 @@ add_hipcc_test(TestFloat128Macros.hip TEST_NAME TestFloat128gnupp14
 
 add_hipcc_test(TestDoubleShuffleOverloads.hip HIPCC_OPTIONS -fsyntax-only)
 add_hipcc_test(TestHipComplexInclude.hip HIPCC_OPTIONS)
+
+add_hipcc_test(TestHipccAcceptCcFiles.cc HIPCC_OPTIONS)
+add_hipcc_test(TestHipccAcceptCppFiles.cpp HIPCC_OPTIONS)
+
+add_test(NAME "TestHipccNeedsDashO"
```

Comment:
Isn't this same as writing `add_hipcc_test(TestHipccNeedDashO.cc)`?

Reply:
no since `add_hipcc_test` macro insers `-o`, the omission of which we are testing.